stdlib/email/src/main/ballerina/src/email/smtp_client_endpoint.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents an SMTP Client, which interacts with an SMTP Server.
+public type SmtpClient client object {
+
+    # Gets invoked during object initialization.
+    #
+    # + clientConfig - Configurations for SMTP Client
+    public function __init(SmtpConfig clientConfig) {
+        error? response = initSmtpClientEndpoint(self, clientConfig);
```

Comment:
Same comment as above re: ignoring errors.

Reply:
Made it to return error type.